pages/en/hosted-service/migrating-subgraph.mdx

```diff
@@ -64,6 +71,8 @@ graph codegen && graph build
 
 And that's it! After you are done publishing, you'll be able to view your subgraphs live on the network via [The Graph Explorer](https://thegraph.com/explorer).
 
+Feel free to leverage the Curation Station (Unofficial) [Discord](https://discord.gg/GVfH3PgANH) and [Telegram](https://t.me/+RqzR_Y_a5eY2Zjdh) to let Curators know that your subgraph is ready to be signaled. It would also be helpful if you share your expected query volume with them.
```

Comment:
These are unofficial sources and we should not reference them on the public docs. But a paragraph that explains the general process of advertising a subgraph is definitely good. Main place for such thing is the official The Graph Discord: thegraph.com/discord